agent.go

```diff
@@ -416,7 +508,10 @@ var initRootfsMounts = []initMount{
 // initAgentAsInit will do the initializations such as setting up the rootfs
 // when this agent has been run as the init process.
 func initAgentAsInit() error {
-	fmt.Printf("initAgentAsInit(), agent version: %s\n", version)
+	err := announce()
```

Comment:
This will fail because `/proc` is not mounted yet.

Reply:
Good catch @bergwolf! I've moved the call to after the mounts now. However, that might not be what you want since you won't get the version details displayed if a mount fails?

Reply:
If a mount fails at this stage, it is mostly unrelated to the agent anyway, -- most likely a kernel bug or some kernel features are not enabled. We can get related info from runtime then. I can live with it.

Reply:
ok - thanks @bergwolf.